src/main/scala/io/pinecone/spark/pinecone/PineconeDataWriter.scala

```diff
@@ -39,9 +39,10 @@ case class PineconeDataWriter(
     try {
       val id = record.getUTF8String(0).toString
       val namespace = record.getUTF8String(1).toString
-      val values = record.getArray(2).toFloatArray().map(float2Float).toIterable.asJava
+      val values = record.getArray(2).toFloatArray().map(float2Float).toIterable
```

Comment:
Just to confirm, the `.asJava` call for the iterable was moved further down to lv 56?

Reply:
Correct!